kernel/src/hil/temp.rs

```diff
@@ -0,0 +1,10 @@
+use returncode::ReturnCode;
```

Comment:
High level, we can maybe elide this HIL for now. We almost certainly want one to consolidate the interfaces for this controller as well as the TMP006 and SI7021 drivers. However, that might be best saved for a different PR.